app/assets/javascripts/comments.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
Is it bad to commit unused files that rails creates unless you need to use them later or should you always delete them and not commit them??